libs/redis/langchain_redis/vectorstores.py

```diff
@@ -739,6 +740,47 @@ def delete(self, ids: Optional[List[str]] = None, **kwargs: Any) -> Optional[boo
             return self._index.drop_keys(keys) == len(ids)
         else:
             return False
+        
+    def get_by_ids(self, ids: Sequence[str], /) -> List[Document]:
+        """Get documents by their IDs.
+
+        The returned documents are expected to have the ID field set to the ID of the
+        document in the vector store.
+
+        Fewer documents may be returned than requested if some IDs are not found or
+        if there are duplicated IDs.
+
+        Users should not assume that the order of the returned documents matches
+        the order of the input IDs. Instead, users should rely on the ID field of the
+        returned documents.
+
+        This method should **NOT** raise exceptions if no documents are found for
+        some IDs.
+
+        Args:
+            ids: List of ids to retrieve.
+
+        Returns:
+            List of Documents.
+        """
+
+        result_docs = []
+        for key in ids:
+            doc_dict = self.index.client.hgetall(
```

Comment:
Should use a pipeline operation for this with `transaciton=False`

Reply:
@tylerhutcherson What do you think about these changes?